eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -311,6 +311,6 @@ page at http://checkstyle.sourceforge.net/config.html -->
 
     <!--CUSTOM CHECKS-->
     <!-- Must use 'logger.logAndThrow' but not directly calling 'throw exception' -->
-<!--    <module name="com.azure.tools.checkstyle.checks.ThrownClientLoggerCheck"/>-->
+    <module name="com.azure.tools.checkstyle.checks.ThrownClientLoggerCheck"/>
```

Comment:
Need to change this to the new class name, this is why all the analyze steps are failing.